shared/.flowconfig

```diff
@@ -42,6 +42,7 @@ esproposal.class_static_fields=enable
 esproposal.class_instance_fields=enable
 experimental.const_params=true
 esproposal.optional_chaining=enable
+merge_timeout=180
```

Comment:
what's this for?

Reply:
>also increased merge_timeout in flowconfig because it's impossible to work with flow on Windows otherwise
Basically the codebase is so massive than on systems with slow IO (ie. WSL) flow won't ever start as the "merge" process timeouts after ~115s. After I did this, it seems to be working fine, although that was enough to convince me to move to Linux.